content/python/concepts/loops/loops.md

```diff
@@ -81,6 +81,30 @@ for group in groups:
     print(name)
 ```
 
+## Do While Loop
+
+A `do while` loop is a control structure that executes a block of code at least once, and then repeatedly executes it as long as a specified condition is met.
```

Comment:
There should be some acknowledgement that this isn't implemented in python. The pseudo code used below actually doesn't apply. It should be modified to represent how a `while` loop can be modified to simulate a do while in other languages.